tests/models/plants/test_stoichiometry.py

```diff
@@ -0,0 +1,437 @@
+import numpy as np  # noqa: D100
+import pytest
+
+from virtual_ecosystem.models.plants.stoichiometry import (
+    FoliageTissue,
+    ReproductiveTissue,
+    RootTissue,
+    StemStoichiometry,
+    WoodTissue,
+)
+
+
+class DummyStemAllometry:
+    """Mock class for stem allometry with predefined masses for testing."""
+
+    foliage_mass = np.array([50.0, 80.0])
+    stem_mass = np.array([100.0, 150.0])
+    reproductive_tissue_mass = np.array([20.0, 40.0])
+
+
+class DummyStemTraits:
+    """Mock class for stem traits with predefined stoichiometric ratios for testing."""
+
+    zeta = 0.1
+    sla = 2.0
+    p_foliage_for_reproductive_tissue = 0.5
+
+
+class DummyExtraTraitsPFT:
+    """Minimal mock for extra PFT traits."""
+
+    pass
+
+
+class DummyCommunity:
+    """Mock class for a plant community with two cohorts for testing."""
+
+    def __init__(self):
+        self.stem_allometry = DummyStemAllometry()
+        self.stem_traits = DummyStemTraits()
+        self.n_cohorts = 2
+        self.cohorts = type("Cohorts", (), {"pft_names": ["pft1", "pft2"]})
+
+
+class DummyAllocation:
+    """Mock class for allocation with predefined values for testing."""
+
+    def __init__(self):
+        self.delta_foliage_mass = np.array([20.0, 10.0])
+        self.delta_stem_mass = np.array([10.0, 5.0])
+        self.foliage_turnover = np.array([30.0, 15.0])
+        self.reproductive_tissue_turnover = np.array([5.0, 2.0])
+        self.fine_root_turnover = np.array([3.0, 1.5])
+
+
+dummy_community = DummyCommunity()
+dummy_alloc = DummyAllocation()
+
+
+def make_stem_stoichiometry():
+    """Helper function to create a StemStoichiometry instance with all tissues."""
+
+    """"""
```

Comment:
I think this is cleaner as an explicit fixture 

```suggestion
@pytest.fixture
def stem_stoichiometry():
    """Helper function to create a StemStoichiometry instance with all tissues."""

    """"""
```